src/components/asthma/components/AsthmaProviderReport/AsthmaProviderReport.tsx

```diff
@@ -247,137 +247,134 @@ export default function (props: AsthmaProviderReportProps) {
+                        {renderStat(language('asthma-provider-report-work'), highWorkAqiDays > 0 ? highWorkAqiDays : language('asthma-provider-report-none'), highWorkAqiDays === 1 ? language('asthma-provider-report-day') : highWorkAqiDays > 1 ? language('asthma-provider-report-days') : '')}
+                        <div style={{ fontSize: '12px', marginTop: '12px' }}>{language('asthma-provider-report-source')}</div>
+                        <div style={{ fontSize: '12px', marginTop: '12px' }}>Source: airnow.gov</div>
                     </div>
                 </div>
-                <div style={{position: 'absolute', bottom: 64, left: 48, right: 48, fontSize: '17px'}}>
-                    <div style={{fontSize: '16px', fontWeight: 700, marginBottom: '16px'}}>For Providers - About this tool & report:</div>
-                    <div style={{marginBottom: '16px'}}>
-                        This report was generated from what is logged in the Asthma Tool, powered by <a href="https://careevolution.com/mydatahelps/" target="_blank">MyDataHelps</a>.
-                        The Asthma Tool enables patients to track daily their symptoms and triggers, provides clear summaries or what they log and surfaces relevant
-                        educational content and resource links. it summarizes the user's symptom logs. This digital tool design was based on promoting self-regulation,
-                        which <a href="https://pubmed.ncbi.nlm.nih.gov/26252889/" target="_blank">studies</a> demonstrate can improve conditions such as asthma. The Asthma Tool does
-                        not recommend any changes in management or provide diagnosis.
+                <div style={{ position: 'absolute', bottom: 64, left: 48, right: 48, fontSize: '17px' }}>
+                    <div style={{ fontSize: '16px', fontWeight: 700, marginBottom: '16px' }}>{language('asthma-provider-report-about-tool')}</div>
+                    <div style={{ marginBottom: '16px' }}>
+                        {language('asthma-provider-report-about-text-part1')}
+                        <a href="https://careevolution.com/mydatahelps/" target="_blank">MyDataHelps</a>
+                        {language('asthma-provider-report-about-text-part2')}
+                        <a href="https://pubmed.ncbi.nlm.nih.gov/26252889/" target="_blank">{language('asthma-provider-report-studies')}</a>
+                        {language('asthma-provider-report-about-text-part3')}
                     </div>
-                    <div style={{marginBottom: '16px'}}>
-                        To provide feedback on this report or tool, please e-mail us: <a href="mailto:asthma@careevolution.com" target="_blank">asthma@careevolution.com</a>
+                    <div style={{ marginBottom: '16px' }}>
+                        {language('asthma-provider-report-feedback-part1')}
```

Comment:
The email needs to be embedded in the prompt text. It won't necessarily appear at the end of the string in all languages. Then you can remove "part1" from the name.

Reply:
@nathia This one is still outstanding.